test/index.js

```diff
@@ -414,6 +414,21 @@ describe('Public API', () => {
 
     });
 
+    describe('isRuntimeConfigured', () => {
+
+        it ('should return true if the runtime environment has been configured', () => {
+            const returnedValue = api.isRuntimeConfigured();
+            expect(returnedValue).to.be.true;
+        });
+
+        it ('should return false if the runtime environment has been configured', () => {
```

Comment:
`has been configured` -> `has not been configured` ?

Reply:
indeed, bad copy paste